packages/next-codemod/transforms/revalidate-to-expire.ts

```diff
@@ -0,0 +1,51 @@
+import type { API, FileInfo } from 'jscodeshift'
+import { createParserFromPath } from '../lib/parser'
+
+export default function transformer(file: FileInfo, _api: API) {
+  const j = createParserFromPath(file.path)
+  const root = j(file.source)
+
+  if (!root.length) {
+    return
+  }
+
+  const imports = root
```

Comment:
I don't think this assignment works since the end result is a `forEach`

Reply:
Good catch, but huh, `forEach` actually returns the collection, so it did work. 🫢
Will modify to `.map` for better understanding!